Planka install issue

### ✅ Have you read and understood the above guidelines?

Yes

### 📜 What is the name of the script you are using?

PLANKA

### 📂 What was the exact command used to execute the script?

bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/ct/planka.sh)"

### ⚙️ What settings are you using?

- [x] Default Settings
- [ ] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 12

### 📝 Provide a clear and concise description of the issue.

Just tried install planka on my proxmox and I have TypeError: Invalid URL (postgresql url).



### 🔄 Steps to reproduce the issue.

Run install script with verbose

### ❌ Paste the full error output (if available).

 ⠦Installing collected packages: urllib3, PyYAML, oauthlib, markdown, idna, click, charset_normalizer, certifi, requests, requests-oauthlib, apprise
 ⠴Successfully installed PyYAML-6.0.2 apprise-1.9.3 certifi-2025.6.15 charset_normalizer-3.4.2 click-8.2.1 idna-3.10 markdown-3.8.2 oauthlib-3.3.0 requests-2.32.4 requests-oauthlib-2.0.0 urllib3-2.5.0

added 891 packages in 28s

160 packages are looking for funding
  run `npm fund` for details
npm notice
npm notice New major version of npm available! 10.9.2 -> 11.4.2
npm notice Changelog: https://github.com/npm/cli/releases/tag/v11.4.2
npm notice To update run: npm install -g npm@11.4.2
npm notice
 ⠦
> db:init
> node db/init.js

 ⠇node:internal/url:818
      href = bindingUrl.parse(input, base, true);
                        ^

TypeError: Invalid URL
    at new URL (node:internal/url:818:25)
    at parse (/opt/planka/planka/node_modules/pg-connection-string/index.js:29:14)
    at new ConnectionParameters (/opt/planka/planka/node_modules/pg/lib/connection-parameters.js:56:42)
    at new Client (/opt/planka/planka/node_modules/pg/lib/client.js:18:33)
    at Client_PG._acquireOnlyConnection (/opt/planka/planka/node_modules/knex/lib/dialects/postgres/index.js:81:24)
    at Client_PG.acquireRawConnection (/opt/planka/planka/node_modules/knex/lib/dialects/postgres/index.js:99:17)
    at create (/opt/planka/planka/node_modules/knex/lib/client.js:262:39) {
  code: 'ERR_INVALID_URL',
  input: 'postgresql://planka:I9m5DsWlDt/8Z@localhost/planka',
  base: 'postgres://base'
}

Node.js v22.16.0
 ⠏
[ERROR] in line 54: exit code 0: while executing command $STD npm run db:init

### 🖼️ Additional context (optional).

_No response_

--- a/install/planka-install.sh
+++ b/install/planka-install.sh
@@ -26,7 +26,7 @@ PG_VERSION="16" setup_postgresql
 msg_info "Setting up PostgreSQL Database"
 DB_NAME=planka
 DB_USER=planka
-DB_PASS="$(openssl rand -base64 18 | cut -c1-13)"
+DB_PASS=$(openssl rand -base64 16 | tr -d '/+=')
 $STD sudo -u postgres psql -c "CREATE ROLE $DB_USER WITH LOGIN PASSWORD '$DB_PASS';"
 $STD sudo -u postgres psql -c "CREATE DATABASE $DB_NAME WITH OWNER $DB_USER ENCODING 'UTF8' TEMPLATE template0;"
 $STD sudo -u postgres psql -c "ALTER ROLE $DB_USER SET client_encoding TO 'utf8';"

